default/source/AssetPaths.hx

```diff
@@ -1,4 +1,100 @@
 package;
 
+/**
+	Helper class to access asset paths in a type-safe manner. 
+
+	`AssetPaths`'s static fields are autogenerated by a neat [Haxe macro](http://haxe.org/manual/macro.html) 
+	from the contents of Project.xml's `<assets>` tag, so you can easily reference them in your code:
+
+	```haxe
+	// Play sound effect using AssetPaths
+	FlxG.sound.play(AssetPaths.mySound__wav);
+	// instead of using string path
+	FlxG.sound.play("assets/sounds/mySound.wav");
+	```
+
+	Using `AssetPaths` make it safe to move(or rename) your assets - whenever that happens
+	you are forced to fix, e.g. `AssetPaths.mySound__wav`, since `mySound__wav` is
+	no longer there and it is a compilation error. What if in a similar situation you used plain strings?
+	If you are lucky you will quickly get a runtime error (in debug mode). Otherwise finding why your
+	game crashes might not be easy.
```

Comment:
The wording here could be better
```suggestion
	AssetPaths values will change whenever you add, remove, rename or move a file. If you remove a file that is still referenced via AssetPaths you'll get a compile error, this could be handy compared to using string paths, which only cause a runtime error if the file is missing.
```